plugins/module_utils/network/edgeos/edgeos.py

```diff
@@ -117,8 +117,7 @@ def load_config(module, commands, commit=False, comment=None):
         out = to_text(out, errors='surrogate_or_strict')
 
         if not out.startswith('No changes'):
-            out = connection.get('show')
-            diff = to_text(out, errors='surrogate_or_strict').strip()
+            diff = out.strip()
```

Comment:
Does this (without the `to_text`) work with both Python 2 and Python 3?

Reply:
I think so, to_text is called on 'out' already on line 117 though I could try and be more explicit (and not reuse the out variable) if you would prefer?

Reply:
Ah right. I missed that :) It's perfect this way, then.